docs/built_in_extensions.rst

```diff
@@ -108,6 +108,37 @@ The HTTP entrypoint is built on top of `werkzeug <http://werkzeug.pocoo.org/>`_.
     Date: Fri, 13 Feb 2015 14:58:48 GMT
 
 
+Expected exceptions can be added to ``expected_exceptions``. This will result in 400 HTTP error to be returned to the client. You can also change exception's formatting by overriding :meth:`~nameko.web.HttpRequestHandler.response_from_exception`:
```

Comment:
agree that the name isn't great - we should consider alternatives

also, note that for for the rpc entrypoint, the concept of this set of exceptions was purely internal to the service, whereas for the http, we are making it part of the service api (since it affects the http status; something we don't have for rpc atm)